holoviews/plotting/bokeh/callbacks.py

```diff
@@ -1191,13 +1191,24 @@ def _watch_position(self):
               }
               if (!xs || !ys) { return }
               for (const i of indices) {
-                const tx = xs[i]
+                let ix = xs[i]
+                let iy = ys[i]
+                if (typeof ix === 'number') {
+                  ix = [ix]
+                  iy = [iy]
+                } else {
+                  while (ix.length && (typeof ix[0] !== 'number')) {
+                    ix = ix[0]
+                    iy = iy[0]
+                  }
+                }
+                const tx = Math.max(...ix)
                 if (!x || (tx > x)) {
-                  x = xs[i]
+                  x = tx
                 }
-                const ty = ys[i]
+                const ty = Math.max(...iy)
```

Comment:
Can't we do the `Math.max` after the while loop and avoid the weird workaround if `ix` is "just" a number?

Reply:
Why "weird" and no I don't see how that fixes anything?

Reply:
To elaborate, `Math.max(...ix)` will not work on a number because it's not iterable.